.github/workflows/test.yml

```diff
@@ -15,10 +15,3 @@ jobs:
           npm install
       - run: |
           npm run all
-  test: # make sure the action works on a clean machine without building
```

Comment:
Name of the file could be 'build-test', no?

Reply:
Renamed the filed to build.yml, since we only build here now.